### ISSUE TITLE
Hangs for a while at start

### ISSUE THREAD

**Issue by freethlua** [freethlua/individualogist-whats-your-archetype#42]
Since I've switched to parsing the transcripts with mustache on client, it takes a bit of processing at the beginning. 

Loading facebook comments also takes a considerable processing and so can be delayed since it's at the bottom anyways. This should probably solve.

Low priority for now, since I think this is probably only really noticeable on slower devices.

**Comment by wilsonljx** [comment]
Hi Freeth,

May I know if you are still checking on what needs to be done? 

Looking forward to hear from you

**Comment by freethlua** [comment]
This is no longer an issue after #44 (comment load time has been reduced) Also the idea of "delaying" isn't needed (and it would've been tricky to implement and may have had it's own performance and reliability cost (monitoring window scroll))
